funsor/pyro/convert.py

```diff
@@ -82,12 +82,12 @@ def tensor_to_funsor(tensor, event_inputs=(), event_output=0, dtype="real"):
 
 def funsor_to_tensor(funsor_, ndims, event_inputs=()):
     """
-    Convert a :class:`funsor.torch.Tensor` to a :class:`torch.Tensor` .
+    Convert a :class:`funsor.tensor.Tensor` to a :class:`torch.Tensor` .
```

Comment:
I guess we'll eventually want to use this same helper for `np.array`, so we can convert to a raw `numeric_array` of underlying type?

Reply:
Yes, I think we should move these utilities to e.g. `tensor.py` file. I would like to defer this to the relevant PR (where we can discuss which file those utilities should be in).